.github/workflows/build.yml

```diff
@@ -23,7 +23,8 @@ jobs:
         config: [macos, linux, windows-lcow, windows-wcow]
         include:
           - config: macos
-            os: macos
+            # since macos-14 the latest runner is arm64
+            os: macos-arm64
```

Comment:
Can we be certain that we'll always land on a `macos-14` runner? Is it deterministic?

Reply:
well, until the latest runner is updated I think the answer is yes.